odxtools/complexcomparam.py

```diff
@@ -49,37 +49,30 @@ def from_et(et_element: ElementTree.Element,
         # default values. Due to the quirky way this is defined by the
         # ODX specification, this is a *major* pain in the butt!
         subparams: NamedItemList[BaseComparam] = NamedItemList()
-        elem_it = iter(et_element)
+        elems = list(iter(et_element))
```

Comment:
I believe the `iter()` is unnecessary here:
```suggestion
        elems = list(et_element)
````